resources/metakey.py

```diff
@@ -377,6 +420,14 @@ def delete(self, key, group_name):
             404:
                 description: When attribute key or group or group permission doesn't exist
         """
+        schema = MetakeyPermissionSetRequestArgsSchema()
+        args_obj = schema.load({
+            "key": key,
+            "group_name": group_name
+        })
+        if args_obj.errors:
+            return {"errors": args_obj.errors}, 400
+
         group = db.session.query(Group).filter(Group.name == group_name).first()
```

Comment:
OTOH here it was originally fetched from the request :thinking:

Reply:
Yup, that was the issue fixed by this PR :/  `group_name` was passed both in requestBody and path. It breaks the single source of ~~truth~~ argument rule.